bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -62,8 +62,46 @@ spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
-      {{- if .Values.initContainers }}
-      initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+      {{- if or .Values.initContainers .Values.auth.tls.enabled }}
+      initContainers: 
+        {{- if .Values.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+        {{- if .Values.auth.tls.enabled }}
+        {{- $fullname := include "keycloak.fullname" . }}
+        - name: init-certs
+          image: {{ include "keycloak.auth.tls.image" . }}
```

Comment:
This helper is missing from `_helpers.tlp`. We should include the following:

```
{{/*
Return the proper Auth TLS image name
*/}}
{{- define "keycloak.auth.tls.image" -}}
{{- include "common.images.image" (dict "imageRoot" .Values.auth.tls.image "global" .Values.global) -}}
{{- end -}}
```

Reply:
Thank you for pointing that out. This has been comitted.